Gruntfile.js

```diff
@@ -81,84 +77,14 @@ module.exports = function(grunt) {
                         // search: 'narrativeMain',
                         search: 'narrativeMain.js',
 
-                        replace: function(match) {
+                        replace: function() {
                             return 'kbase-narrative-min.js';
                         },
                         flags: ''
                     }
                 ]
             }
         },
-
-        // Testing with Karma!
-        karma: {
-            unit: {
-                configFile: 'test/unit/karma.conf.js',
-            },
-            dev: {
-                // to do - add watch here
-                configFile: 'test/unit/karma.conf.js',
-                reporters: ['progress', 'coverage'],
-                coverageReporter: {
-                    dir: 'build/test-coverage/',
-                    reporters: [
-                        { type: 'html', subdir: 'html' },
-                    ],
-                },
-
-                autoWatch: true,
-                singleRun: false,
-            }
-        },
-
-        jasmine_nodejs: {
```

Comment:
running this does precisely nothing; `grunt-jasmine-nodejs` has been deprecated (in favour of using a npm script) so there is little point in keeping this code around.